### PR TITLE
Add new yast lan cli test

### PR DESCRIPTION
**Pull request by vpelcak**
New test poo#49250: tests add, edit, delete, list and status of network
interfaces.

- Related ticket: https://progress.opensuse.org/issues/49250
- Needles: none
- Verification run: http://deva.suse.cz/tests/35

### CODE REVIEW

File: tests/yast2_cmd/yast_lan.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
+ # SUSE's openQA tests
+ #
+ # Copyright © 2009-2013 Bernhard M. Wiedemann
+ # Copyright © 2012-2019 SUSE LLC
```
Comment (mimi1vx) [inline tests/yast2_cmd/yast_lan.pm]: not needed coputight from 2009 to 2019 in new test 
simple 2019 SUSE LLC is sufficient

File: tests/yast2_cmd/yast_lan.pm [hunk 2]
```diff
@@ -0,0 +1,30 @@
+ # SUSE's openQA tests
```
Comment (mimi1vx) [inline tests/yast2_cmd/yast_lan.pm]: whole header is badly indented

File: tests/yast2_cmd/yast_lan.pm [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    assert_script_run 'zypper -n in yast2-network';
+    assert_script_run 'yast lan add name=vlan50 ethdevice=eth0';
+    assert_script_run 'yast lan list';
+    assert_script_run 'yast lan show id=1';
+    assert_script_run 'yast lan edit id=1 bootproto=dhcp';
+    assert_script_run 'yast lan show id=1';
+    assert_script_run 'yast lan delete id=1';
+    assert_script_run 'yast lan list';
+}
+sub post_run_hook {}
```
Comment (mimi1vx) [inline tests/yast2_cmd/yast_lan.pm]: Why ? remove this line

File: tests/yast2_cmd/yast_lan.pm [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ # Maintainer: Vit Pelcak <vpelcak@suse.cz>
+
+use base 'consoletest';
+use strict;
+use testapi;
+sub run {
+    select_console 'root-console';
+    assert_script_run 'zypper -n in yast2-network';
+    assert_script_run 'yast lan add name=vlan50 ethdevice=eth0';
+    assert_script_run 'yast lan list';
```
Comment (mimi1vx) [inline tests/yast2_cmd/yast_lan.pm]: maybe not only check if the command returns 0 but also excepted the output

Comment (mimi1vx) [reply tests/yast2_cmd/yast_lan.pm]: see http://open.qa/api/testapi/#_validate_script_output

File: tests/yast2_cmd/yast_lan.pm (hunk 1 above)
Comment (vpelcak) [reply tests/yast2_cmd/yast_lan.pm]: Fixed

File: tests/yast2_cmd/yast_lan.pm (hunk 3 above)
Comment (vpelcak) [reply tests/yast2_cmd/yast_lan.pm]: Fixed

File: tests/yast2_cmd/yast_lan.pm (hunk 2 above)
Comment (vpelcak) [reply tests/yast2_cmd/yast_lan.pm]: Fixed

File: tests/yast2_cmd/yast_lan.pm (hunk 4 above)
Comment (vpelcak) [reply tests/yast2_cmd/yast_lan.pm]: Fixed

Comment (mimi1vx) [reply tests/yast2_cmd/yast_lan.pm]: how ? I dont see any change

Comment (vpelcak) [reply tests/yast2_cmd/yast_lan.pm]: Fixed now.
For yast lan edit I had to increase timeout to 60s as otherwise the test timed out.

**Pull request merged**